### PR TITLE
Update SNS intro on the developer doc side

### PR DESCRIPTION
**Pull request by LaraAS**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/sns/sns-intro.md [hunk 1]
```diff
@@ -239,14 +148,14 @@ The SNS communities are responsible for individually topping up the cycles of
 all SNS canisters as well as all dapp canisters that are controlled by the SNS.
 Special care must be taken that cycles are also monitored for canisters that
 are automatically created. In particular, this includes the archive canisters
-that are spawn by the ledger canister.
+that are automatically spawn by the ledger canister.
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/sns-intro.md:151]:
```suggestion
that are automatically spawned by the ledger canister.
```

File: docs/developer-docs/integrations/sns/sns-intro.md [hunk 2]
```diff
@@ -239,14 +148,14 @@ The SNS communities are responsible for individually topping up the cycles of
...
 **If the archive canisters are not provided
 with sufficient cycles, the ledger block history may be lost.**
 :::
 In the future, cycles management will be simplified in a new feature that 
 allows canister groups, where cycles can be managed across different canisters.
 
-There are separate pages where you can fine more details regarding [how 
+There are separate pages where you can find more details regarding [how
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/sns-intro.md:158]: TODO: check in deployment preview if links that span multiple lines work

**Review by sesi200 — APPROVED** [review]

File: docs/developer-docs/integrations/sns/sns-intro.md (hunk 2 above)
Comment (LaraAS) [reply docs/developer-docs/integrations/sns/sns-intro.md:158]: typo is not here anymore. 
Noted to check

Comment (aisconnolly) [reply docs/developer-docs/integrations/sns/sns-intro.md:156]: @LaraAS @sesi200 Links seem fine: https://deploy-preview-560--icportal.netlify.app/docs/current/developer-docs/integrations/sns/sns-intro/

Comment (LaraAS) [reply docs/developer-docs/integrations/sns/sns-intro.md:156]: Thanks for checking. Looks good indeed!

**Pull request merged**